webapp/src/container.tsx

```diff
@@ -66,6 +77,7 @@ export class DocsMenu extends data.PureComponent<DocsMenuProps, {}> {
         const targetTheme = pxt.appTarget.appTheme;
         return <sui.DropdownMenu role="menuitem" icon="help circle large"
             className="item mobile hide help-dropdown-menuitem" textClass={"landscape only"} title={lf("Help")} >
+            {(this.props.editor === "Blocks") && getTourItem(parent)}
```

Comment:
Should I just add a check here for the target?
```suggestion
            {(this.props.editor === "Blocks") && (pxt.appTarget.id === "microbit") && getTourItem(parent)}
```

Reply:
Add a feature flag to microbit's pxtarget.json.

Reply:
Do you have an example I can reference?

Reply:
+1 Something like this: https://github.com/microsoft/pxt-microbit/blob/master/pxtarget.json#L590
Then you can enable this for any target and see how it works. The feature becomes feature flag driven instead of target driven.

Reply:
In code, feature flags are defined in the `AppTheme` interface: https://github.com/microsoft/pxt/blob/a44f003cb5d80c5fd9a77c09659cc0c3c378336b/localtypings/pxtarget.d.ts#L287

At runtime this interface is instantiated as `pxt.appTarget.appTheme`. Search for that to see how it is used to control feature UI. Here's an example of choosing to show immersive reader based on the feature flag: https://github.com/microsoft/pxt/blob/a44f003cb5d80c5fd9a77c09659cc0c3c378336b/webapp/src/components/tutorial/TutorialContainer.tsx#L50

Reply:
Thanks!